### ISSUE TITLE
Exit Path - visual issues and memory issues

### ISSUE THREAD

**Issue by sodscg** [ruffle-rs/ruffle#8856]
### Describe the bug

`ERROR ruffle_core::display_object::movie_clip] Error occured when running AVM2 frame script: RustError("Non-public property http://adobe.com/AS3/2006/builtin::toString not found on Object")`

### Expected behavior

the game should play properly

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

[538792_Exit.zip](https://github.com/ruffle-rs/ruffle/files/10262071/538792_Exit.zip)

**Comment by Lord-McSweeney** [comment]
The armor games logo renders incorrectly in this SWF (pure white). The game now plays until the third level, where there are several "Display object container has no child with name " warnings and a white screen.

**Comment by sodscg** [comment]
this game seems to work, but now crashes with 'out of memory' after test lab 4

**Comment by TheGentleChainsaw** [comment]
Just playtested the web version [over on Newgrounds](https://www.newgrounds.com/portal/view/538792?emulate=flash) and I can confirm that, even though some parts of the game are still quite laggy, _especially_ the VHS rewind effect when you die, I was able to complete the game without any out-of-memory errors. It did take a while for the game to load in each new level, though. I think it's worth mentioning that the game performs significantly better in the desktop player.

Apart from the aforementioned issues of lag and loading times in the web player, the only other problem I've noticed is a small rendering error that causes a lack of glowing effects and thereby makes the intro animation disappear into the black background. See the embedded video (first capture is from Ruffle desktop, second capture is from Adobe Flash Player):

https://user-images.githubusercontent.com/96608266/226601767-8e8b5d4c-d43e-4422-b290-abc570d1747b.mp4
